### DISCUSSION TITLE
How to open new terminal tab/window

### DISCUSSION THREAD

**Discussion by JCodeX-Dev** [spring-projects/spring-shell#698] [Q&A]
Hey, I am trying create an application which requires opening multiple terminal tabs/windows without losing access to previous terminal, i.e., they all run in parallel with different context.
https://stackoverflow.com/questions/75899537/how-to-open-a-new-terminal-tab-using-spring-boot-dependency-spring-shell

**Comment by jvalkeal** [comment]
That's a role of your terminal emulator you use, nothing to do with spring-shell. tmux is also one what people use to have different sessions within one terminal window.

**Reply by JCodeX-Dev** [reply]
I am trying to make a chat application using spring shell, which requires each chat session to run in separate window
